web/src/engine/predictive-text/templates/src/trie-model.ts

```diff
@@ -303,6 +303,18 @@ export default class TrieModel implements LexicalModel {
   }
 
   predict(transform: Transform, context: Context): Distribution<Suggestion> {
+    //#region Helper function
+    function makeDistribution(suggestions: WithOutcome<Suggestion>[]): Distribution<Suggestion> {
```

Comment:
I guess we could make it clearer that this is local by turning it into a const var?

```suggestion
    const makeDistribution = function(suggestions: WithOutcome<Suggestion>[]): Distribution<Suggestion> {
```